### DISCUSSION TITLE
No debug output written

### DISCUSSION THREAD

**Discussion by MMcKester** [ccache/ccache#558] [Support]
Hi,

I am using `ccache 3.7.7` on my macOS machine, installed via `homebrew`. This is my `~/.ccache/cache.conf` file:

```
max_size = 25.0G
debug = true
```

```
$ ccache -p
(default) base_dir = 
(default) cache_dir = /Users/daniel/.ccache
(default) cache_dir_levels = 2
(default) compiler = 
(default) compiler_check = mtime
(default) compression = false
(default) compression_level = 6
(default) cpp_extension = 
(/Users/daniel/.ccache/ccache.conf) debug = true
(default) depend_mode = false
(default) direct_mode = true
(default) disable = false
(default) extra_files_to_hash = 
(default) hard_link = false
(default) hash_dir = true
(default) ignore_headers_in_manifest = 
(default) keep_comments_cpp = false
(default) limit_multiple = 0.8
(default) log_file = 
(default) max_files = 0
(/Users/daniel/.ccache/ccache.conf) max_size = 25.0G
(default) path = 
(default) pch_external_checksum = false
(default) prefix_command = 
(default) prefix_command_cpp = 
(default) read_only = false
(default) read_only_direct = false
(default) recache = false
(default) run_second_cpp = true
(default) sloppiness = 
(default) stats = true
(default) temporary_dir = 
(default) umask = 
```

Executing ccache and clang fills the directory where the conf file is located. I can only find some *.manifest files. But I miss the `<objectfile>.ccache-input-c` files. Am I doing something wrong? Thanks!

**Comment by jrosdahl** [comment]
Hi,

> Executing ccache and clang fills the directory where the conf file is located. I can only find some *.manifest files. But I miss the <objectfile>.ccache-input-c files. Am I doing something wrong? Thanks!

The debug files are not written to the ccache directory – they are written next to the object file produced by the compiler. For instance, if you run `ccache gcc -c test.c -o test.o` then the files will be called `test.o.ccache-input-c` etc. in the current working directory.

**Comment by MMcKester** [comment]
Thank you so much!

**Comment by Timmmm** [comment]
Note, small gotcha: this only does something if you are compiling object files (`-c`). I just spent 10 minutes trying to figure out why `CCACHE_DEBUG=1 ccache gcc test.c -o test` didn't write anything.
